### PR TITLE
Added instructions of installing the API using nuget.org

### PR DESCRIPTION
**Pull request by ghost**
The Solace Messaging API for C#/.NET (also referred to as SolClient for .NET) is going to be available through nuget.org.

This change has instructions of installing the API using nuget.org for all currently available C# tutorials.

Addresses issue #11.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,16 +10,34 @@ These tutorials will get you up to speed and sending messages with Solace techno
...
+Use the NuGet console or the NuGet Visual Studio Extension to download the [SolaceSystems.Solclient.Messaging](http://nuget.org/packages/SolaceSystems.Solclient.Messaging/) package for your solution and to install it for your project.
+
+The package contains the required libraries and brief API documentation. It will automatically copy correct libraries from the package to the target directory at build time, but of course if you compile your program from the command line you would need to refer to the API assemblies and libraries locations explicitly.
+
+Notice that in this case both x64 and x86 API assemblies and libraries have the same names.
+
+### Get the API: Using the Solace Developer Portal
+
+The SolClient for .NET can be [downloaded here]({{ site.links-downloads }}){:target="_top"}. That distribution is a zip file containing the required libraries, detailed API documentation, and examples.
+
```
Comment (mdspielman) [inline README.md:30]: The "Download here" link in the above is a broken link in the readme. That only works in Jekyll.

**Review by mdspielman — COMMENTED** [review]
Let's fix up the README typo if we can.

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md:30]: So what do I put in there?

**Comment by ghost** [comment]
What typo, Mark? You mean the invalid download link? What do I put there instead?

File: README.md (hunk 1 above)
Comment (mdspielman) [reply README.md:30]: http://dev.solace.com/downloads/

**Pull request closed without merging**